ase/src/main/java/com/ase/ase/rest/response/TableDataPerDate.java

```diff
@@ -0,0 +1,39 @@
+package com.ase.ase.rest.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class TableDataPerDate {
+
+    private final String date;
+    private final List<HospitalSituationPerDate> situation;
```

Comment:
After SQL Statement and REST call is implemented, change this to int

Reply:
Should be done now, not when the sql changes because it only holds dummy data so it can be done now.